### DISCUSSION TITLE
Pot | 使用指南

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#593] [General]
# Pot | 使用指南

跨平台划词翻译和OCR软件

https://pot-app.com/docs/

**Comment by Dnone1** [comment]
简直了，太好用了，程序员的救星🎈🎈🎈🎈作者真nice

**Comment by Doleres95** [comment]
macos版本现在可以正常使用么，为什么我打开翻译界面是空白，也没有设置界面

**Reply by hapbp** [reply]
我也遇到了这个问题。第一次能打开设置，后面就打不开了

**Comment by syx404** [comment]
在福新pdf上无法划词翻译，请问有解决办法吗？

**Comment by Duo399** [comment]
mac的interl的版本一到截屏翻译就白屏，不知道怎么解决比较合适

**Comment by syx404** [comment]
如果能够支持划线自动翻译就好了，而不需要按下快捷键，这样会提供更多的方方便。

**Comment by SWeeTvT** [comment]
用不了，隔三差五就失效，无论翻墙还是不翻墙，翻译的时候要么半天无响应，要么报错：“{"error_code":"54004","error_msg":"Please recharge"}”

**Comment by Awuluhe** [comment]
翻译能支持使用历史记录吗，翻译相同内容直接显示，不想再等重新翻译了
